pages/api/events/class/index.ts

```diff
@@ -57,28 +87,81 @@ const classEventHandler: NextApiHandler = async (req: NextApiRequest, res: NextA
               minute: startTime.minute,
               second: startTime.second,
             })
-            .setZone(newEvent.timeZone);
+            .setZone(newEvent.timeZone, { keepLocalTime: true });
```

Comment:
good catch for this. I changed it most palces in the app, but forgot about it here :)